gulp-core-build-sass/src/SassTask.ts

```diff
@@ -72,15 +79,26 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
     }));
 
     const srcPattern: string[] = this.taskConfig.sassMatch.slice(0);
-    const moduleSrcPattern: string[] = srcPattern.map((value: string) => value.replace('.scss', '.module.scss'));
+
+    const checkFilenameForCSSModule: (file: gulpUtil.File) => void = (file: gulpUtil.File) => {
+      if (!path.basename(file.path).match(/module\.scss$/)) {
+
+        const filepath: string = path.relative(this.buildConfig.rootPath, file.path);
+        this.logWarning(`${filepath}: filename should end with module.scss`);
+      }
+    };
+
+    console.log('warnOnNonCSSModules: ' + this.taskConfig.warnOnNonCSSModules);
```

Comment:
Should you remove this console.log?

You can use `this.logVerbose` I believe if you need to log verbose stuff.